src/types/generator.ts

```diff
@@ -105,6 +105,7 @@ export type GeneratorOptions = {
   override: NormalizedOverrideOutput;
   context: ContextSpecs;
   mock: boolean;
+  delay?: number;
```

Comment:
I would add the property in `override.mock` what do you think about it?

Reply:
I've added a PR about it
https://github.com/anymaniax/orval/pull/431

Reply:
Thanks @Graveheart